tests/unit_tests/udfs/test_base_udf.py

```diff
@@ -133,8 +134,8 @@ def test_model_downloader_missing_parameters(description, bucketfs_conn_name, bu
     udf.run(mock_ctx)
     res = mock_ctx.output
     error_field = res[0][-1]
-    expected_error_start = f"For each model model_name, bucketfs_conn and sub_dir need to be provided. " \
-                           f"Found model_name = {model_name},"
-    expected_error_end = f" sub_dir = {sub_dir}"
+    pattern = f"For each model model_name, bucketfs_conn and sub_dir need to be provided. Found model_name = " \
+              f"{model_name}, bucketfs_conn = .*, sub_dir = {sub_dir}."
+
+    assert re.search(pattern, error_field)
```

Comment:
I meant something, like that. It gives you a nicer diff

https://github.com/exasol/integration-test-docker-environment/blob/d3450d947ea88e0d4900beac2a2a0cfd65c3806a/test/matchers.py#L4